### PR TITLE
Nipy extra

### PR DESCRIPTION
**Pull request by TheChymera**
Added nipy-data and nipy-template packages

### CODE REVIEW

File: sci-libs/nipy-data/nipy-data-0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+HOMEPAGE="http://nipy.org/"
+SRC_URI="http://nipy.org/data-packages/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86 amd64-linux x86-linux"
+IUSE=""
+
+RDEPEND=""
+DEPEND=""
```
Comment (jlec) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: Drop if empty.

File: sci-libs/nipy-data/nipy-data-0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+inherit distutils-r1 eutils multilib flag-o-matic
+
+DESCRIPTION="Neuroimaging tools for Python"
+HOMEPAGE="http://nipy.org/"
+SRC_URI="http://nipy.org/data-packages/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86 amd64-linux x86-linux"
```
Comment (jlec) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: no stable KEYWORDs.

File: sci-libs/nipy-templates/nipy-templates-0.2.ebuild [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+inherit distutils-r1 eutils multilib flag-o-matic
+
+DESCRIPTION="Neuroimaging tools for Python"
+HOMEPAGE="http://nipy.org/"
+SRC_URI="http://nipy.org/data-packages/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="amd64 x86 amd64-linux x86-linux"
```
Comment (jlec) [inline sci-libs/nipy-templates/nipy-templates-0.2.ebuild]: see above

File: sci-libs/nipy-templates/nipy-templates-0.2.ebuild [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+IUSE=""
+
+RDEPEND=""
+DEPEND=""
```
Comment (jlec) [inline sci-libs/nipy-templates/nipy-templates-0.2.ebuild]: see above

File: sci-libs/nipy-data/nipy-data-0.2.ebuild (hunk 2 above)
Comment (TheChymera) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: It's a colection of rather static example data files. afaict they have not been modified for a long tim, nor will they be modified in the future. There is really nothing about them that can be unstable.

Comment (marbre) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: Packages in the overlay are never masked stable.

**Comment by marbre** [comment]
What about removing sci-libs/nipy? The package is present now twice.

File: sci-libs/nipy-data/nipy-data-0.2.ebuild (hunk 2 above)
Comment (TheChymera) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: Even if the concept of "unstable" does not aply to the kind of content they provide?

Comment (jlec) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: there are no stable packages in the overlay. That's a rule.

**Comment by TheChymera** [comment]
@marbre I thought I had moved it. I'll make a separate PR

**Comment by TheChymera** [comment]
@marbre or no, I'll fix it here-....

File: sci-libs/nipy-templates/nipy-templates-0.2.ebuild [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils multilib flag-o-matic
```
Comment (jlec) [inline sci-libs/nipy-templates/nipy-templates-0.2.ebuild]: Only distutils-r1 used. Please drop the rest

File: sci-libs/nipy-templates/nipy-templates-0.2.ebuild [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+DESCRIPTION="Neuroimaging tools for Python"
```
Comment (jlec) [inline sci-libs/nipy-templates/nipy-templates-0.2.ebuild]: Isn't this package about templates?

File: sci-libs/nipy-data/nipy-data-0.2.ebuild [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils multilib flag-o-matic
+
+DESCRIPTION="Neuroimaging tools for Python"
```
Comment (jlec) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: data??

File: sci-libs/nipy-data/nipy-data-0.2.ebuild [hunk 8]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 eutils multilib flag-o-matic
```
Comment (jlec) [inline sci-libs/nipy-data/nipy-data-0.2.ebuild]: to many eclasses used.

**Comment by jlec** [comment]
@TheChymera Could you please rebase this PR onto latest master?

**Comment by TheChymera** [comment]
@jlec is this better now?

**Comment by SoapGentoo** [comment]
@TheChymera could you possibly squash and rebase on master? The number of commits is staggering, and we prefer to keep our history as linear as possible. If possible, merge all commits to the same ebuild into one.

**Comment by TheChymera** [comment]
@SoapGentoo it was already rebased, now I also followed the instructions [from here](https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/) and squashed the stuff related to nipy-data and nipy-templates.

I hope this is alright now.

File: profiles/updates/4Q-2015 [hunk 9]
```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
 move dev-python/nilearn sci-biology/nilearn
+=======
+move sci-libs/nipy sci-biology/nipy
+>>>>>>> 8101442... added move notice
```
Comment (jlec) [inline profiles/updates/4Q-2015]: This is not correctly merged.

Comment (TheChymera) [inline profiles/updates/4Q-2015]: what can I do about it, just edit it?

Comment (jlec) [inline profiles/updates/4Q-2015]: probably and squash the commit back into the rest.

**Comment by TheChymera** [comment]
@jlec @SoapGentoo ok, I decided to bury this mess and start anew, but I can't seem to be able to clone the master branch from gentoo science, whatever I do `nipy-extra` and `nipy-templates` come back:

```
chymera@zenbookhost ~/src/neurogentoo/sci-libs $ git branch -D sci
Deleted branch sci (was d7b8396).
chymera@zenbookhost ~/src/neurogentoo/sci-libs $ git remote -v
official	https://github.com/gentoo-science/sci.git (fetch)
official	https://github.com/gentoo-science/sci.git (push)
origin	git@github.com:TheChymera/neurogentoo.git (fetch)
origin	git@github.com:TheChymera/neurogentoo.git (push)
chymera@zenbookhost ~/src/neurogentoo/sci-libs $ git fetch official master
From https://github.com/gentoo-science/sci
 * branch            master     -> FETCH_HEAD
chymera@zenbookhost ~/src/neurogentoo/sci-libs $ git branch -f sci official/master
Branch sci set up to track remote branch master from official.
chymera@zenbookhost ~/src/neurogentoo/sci-libs $ ls nipy
nipy/           nipy-data/      nipype/         nipy-templates/ 
```

Any ideas what I can do?

**Comment by SoapGentoo** [comment]
@TheChymera maybe just restart the whole process from scratch, i.e., clone the main science repo, fork it from the command line, set the remote/upstream URLs, then create a new branch. Once you're finished, `push -f` to kill the old stuff

**Pull request closed without merging**